### ISSUE TITLE
Errors in configuration file deletes most configuration information

### ISSUE THREAD

**Issue by goneall** [lfscanning/scaffold#41]
If there is an error in the configuration file, any configuration data after the error is truncated.

This can be easily worked-around by copying the previous version of the configuration file back over.

**Comment by goneall** [comment]
This is caused by the `loadConfig` function simply returning on error which then continues to process and, at times at least, will save the "updated" configuration file.

For example:
```
            cfg._month = config_dict.get('month', "")
            if cfg._month == "":
                print(f'No valid month found in config section')
                return cfg
```

To resolve this, we could throw an exception rather than returning.  This should stop the processing.

**Comment by goneall** [comment]
Resolved with PR #42
